source/adjust.js

```diff
@@ -7,6 +7,8 @@ import _curry3 from './internal/_curry3.js';
  * new copy of the array with the element at the given index replaced with the
  * result of the function application.
  *
+ * When `idx < 0 || idx >= list.length`, the original list is returned.
```

Comment:
This isn't quite right since we handle negative indexes.

"When `idx < -list.length || idx >= list.length`, the original list is returned." or something like that?